docs/general/getting_started.md

```diff
@@ -96,3 +96,12 @@ class PostsController < ApplicationController
 
 end
 ```
+
+If you wish to use Rails url helpers for link generation, e.g., `link(:resources) { resources_url }`, ensure your application sets
+`Rails.application.routes.default_url_options`.
+
```

Comment:
I don't know. I was thinking it would be easy to miss this in several places if it just linked to once section.